transform/mattermost-analytics/models/staging/cws/_cws__models.yml

```diff
@@ -64,3 +64,23 @@ models:
       - name: is_terms_accepted
         description: Whether the user accepted the terms and conditions.
 
+  - name: stg_cws__marketing
```

Comment:
Same issue here: the name is general `stg_cws_marketing` but the content is specific: "Security Newsletter data"

Reply:
Just a note that the convention is `stg_<source name>__<source table name>`. Having said that, it seems that the description should contain more info. It would be nice to capture what's the purpose of this table. Nick should be able to help us with that.

Reply:
yes, need to talk to him about it.
1) If the intention is to capture all types of subscribed content in a generic "marketing" table (and for now we only have `security_newsletter` content, then we should prob. make that distinction on our end - not sure if description is enough - what would be the name of a different model for a non-security_newsletter content?
2) If the intention is to have the marketing table contain only `security_newsletter` content - then we should push for Nick to rename the table imo.

Reply:
@catalintomai option 1 is the intention here - generic marketing table, that for now only contains `security_newsletter`

Reply:
Thank you Nick.